packages/datastore/tests/tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "moduleResolution": "node",
     "target": "ES5",
     "outDir": "./build",
-    "lib": ["ES5", "DOM"],
+    "lib": ["ES6", "DOM"],
```

Comment:
Why this change? We are still compiling down to ES5.

Reply:
This should still compile to ES5, due to the `target` field above, no? I don't recall why I wanted to bump this here (possibly a transient usage of an es6 feature).